tests/core/Jalasoft.TeamUp.Projects.Core.Tests/ProjectsServiceTests.cs

```diff
@@ -132,5 +132,41 @@ public void DeleteProjectById_IdIsValid_ProjectRemoved()
             // Assert
             Assert.NotEqual(countBeforeDelete, countAfterDelete);
         }
+
+        [Fact]
+        public void UpdateProject_Returns_SingleUpdatedProject()
+        {
```

Comment:
Here too  UpdateProjectById_ProjectIsValid_ProjectUpdated()

Reply:
Also updated it